Snakefile

```diff
@@ -485,45 +424,14 @@ rule all_tissue_prediction_signal_removed:
         "--all_tissue "
         "--correction signal "
         "--weighted_loss "
-        "--disable_optuna "
-
-rule transfer_tissue:
-    input:
-        "data/recount_tpm.pkl",
-        imputation_model = "model_configs/imputation/{impute}.yml",
-        dataset_config = "dataset_configs/recount_dataset.yml",
-    threads: 8
-    output:
-        "results/tissue_impute.{impute}_{seed}.tsv"
-    shell:
-        "python saged/imputation_pretraining.py {input.dataset_config} {input.imputation_model} "
-        "results/tissue_impute.{wildcards.impute}_{wildcards.seed}.tsv "
-        "--neptune_config neptune.yml "
-        "--seed {wildcards.seed} "
-        "--weighted_loss "
-
-rule all_tissue_biobert:
-    threads: 16
-    input:
-        "dataset_configs/recount_dataset.yml",
-        "data/recount_embeddings.hdf5",
-        supervised_model = "model_configs/supervised/{supervised}.yml",
-        dataset_config = "dataset_configs/recount_dataset.yml",
-    output:
-        "results/all-tissue-biobert.{supervised}_{seed}.tsv"
-    shell:
-        "python saged/predict_tissue.py {input.dataset_config} {input.supervised_model} "
-        "results/all-tissue-biobert.{wildcards.supervised}_{wildcards.seed}.tsv "
-        "--neptune_config neptune.yml "
-        "--seed {wildcards.seed} "
-        "--all_tissue "
-        "--biobert "
-        "--weighted_loss "
+        # "--disable_optuna "
```

Comment:
Do you want to leave these commented-out lines in? (In other words, is there a case where you might want to run this script with optuna disabled, or will you always run it with optuna at this point?)

Reply:
Or maybe you could add an argument to the whole workflow that determines whether or not to do hyperparameter optimization? (Not sure if this is a thing you can do in Snakemake but my guess is yes?)

Reply:
I think I'll leave them in; there might be a time in the future where I need to run them without optuna. I'm currently planning to remove them some time down the line when I finish cleaning up the Snakefile